.gitignore

```diff
@@ -19,10 +19,13 @@ erl_crash.dump
 # Also ignore archive artifacts (built via "mix archive.build").
 *.ez
 
-protoc-gen-elixir
+# Ignore package tarball (built via "mix hex.build").
+protobuf-*.tar
+
+# Temporary files for e.g. tests.
+/tmp/
```

Comment:
I ran tests and nothing generated anything in `/tmp`. Please remove this.

Reply:
> I ran tests and nothing generated anything in `/tmp`. Please remove this.

Noted. FYI, the `.gitignore` was copied and merged into existing `.gitignore` from generated file from `mix new foobar` using latest Elixir version.